.github/workflows/eden.yml

```diff
@@ -14,140 +15,7 @@ on:  # yamllint disable-line rule:truthy
       - 'docs/**'
 
 jobs:
-  integration:
-    name: Integration test (tpm=${{ matrix.tpm }};${{ matrix.fs }})
-    runs-on: ubuntu-22.04
-    strategy:
-      fail-fast: false
-      matrix:
-        tpm: ["true", "false"]
-        fs: ["zfs", "ext4"]
-    if: ${{ github.event.review.state == 'approved' || github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags') }}
-    steps:
-      - name: Check
-        run: |
-          for addr in $(ip addr list|sed -En -e 's/.*inet ([0-9.]+).*/\1/p')
-          do
-              if echo "$addr" | grep -q -E "10.11.(12|13).[0-9]+"; then
-                echo "$addr overlaps with test"; exit 1
-              fi
-          done
-          sudo df -h
-          sudo swapoff -a
-          sudo free
-      - name: setup
-        run: |
-          sudo add-apt-repository ppa:stefanberger/swtpm-jammy
-          sudo apt install -y qemu-utils qemu-system-x86 jq swtpm
-      - name: get eve
-        uses: actions/checkout@v3
-        with:
-          path: 'eve'
-      - name: prepare eden
-        run: |
-          if [ -f ${{ github.workspace }}/eve/tests/eden/eden-version ]; then
-            EDEN_VERSION=$(cat ${{ github.workspace }}/eve/tests/eden/eden-version)
-          else
-            EDEN_VERSION=lfedge/eden:0.8.0
-          fi
-          docker run -v $PWD:/out $EDEN_VERSION cp -a /eden/. /out/
-          sudo chown -R $(whoami) .
-          ./eden config add default
-      - name: fetch or build eve
-        env:
-          TAG: pr${{ github.event.pull_request.number }}
-          CACHE: evebuild/danger
-        run: |
-          BUILD=true
-          if docker pull "$CACHE:$TAG-kvm"; then
-            # we should check version of pulled image
-            # since in case of build in progress we do not want to test
-            # old version pushed to docker hub
-            # version logic must be aligned with PR build workflow
-            COMMIT_ID=$(git --git-dir ./eve/.git describe --abbrev=8 --always)
-            EXPECTED_VERSION="0.0.0-$TAG-$COMMIT_ID-kvm-amd64"
-            PULLED_VERSION=$(docker run --rm "$CACHE:$TAG-kvm" version)
-            if [ "$PULLED_VERSION" = "$EXPECTED_VERSION" ]; then
-              docker tag "$CACHE:$TAG-kvm" "lfedge/eve:$TAG-kvm"
```

Comment:
Whereas I consider it more of a hack -- tagging local builds as lfedge/eve ones and then using them, inside eden and then overriding environment variable TAG (because eden looks for env variables and sets them). We need to take into consideration what's commented above

Reply:
And perhaps tagging is not a bad idea after all, that way we will not have any problems with evebuild/danger on master